pkgs/sage-conf_pypi/setup.py

```diff
@@ -26,57 +26,82 @@
 class build_py(setuptools_build_py):
 
     def run(self):
+
         HERE = os.path.dirname(__file__)
-        if self.editable_mode:
+
+        SAGE_CONF_FILE = os.environ.get('SAGE_CONF_FILE', None)
+        SAGE_CONF_ENV_FILE = os.environ.get('SAGE_CONF_ENV_FILE', None)
+
+        if SAGE_CONF_FILE or SAGE_CONF_ENV_FILE:
+            try:
+                shutil.copyfile(SAGE_CONF_FILE or '/dev/null',
+                                os.path.join(HERE, '_sage_conf', '_conf.py'))
+            except OSError:
+                raise SetupError(f"environment variable SAGE_CONF_FILE is set to {SAGE_CONF_FILE}, "
+                                 f"but the file cannot be copied")
+            if SAGE_CONF_ENV_FILE and SAGE_CONF_ENV_FILE != '/dev/null':
+                try:
+                    shutil.copyfile(SAGE_CONF_ENV_FILE or '/dev/null',
+                                    os.path.join(HERE, 'bin', 'sage-env-config'))
+                except OSError:
+                    raise SetupError(f"environment variable SAGE_CONF_ENV_FILE is set to {SAGE_CONF_ENV_FILE}, "
+                                     f"but the file cannot be copied")
+            else:
+                # build_scripts skips empty files, so create something nonempty
+                with open(os.path.join(HERE, 'bin', 'sage-env-config'), 'w') as f:
+                    print("# -*- shell-script -*-", file=f)
```

Comment:
I am perfectly happy with it being skipped. I am currently not shipping this file in any way shape or form and everything is fine. The main consumer at runtime is the sage script and it can cope with it not being present https://github.com/sagemath/sage/blob/e417e2205be84d6d567b8897527fa6945ad09bdb/src/bin/sage#L176
So, I'd rather skip it rather than having a token file.

Reply:
The problem is that after skipping the installation, setuptools gives an error because the file is named in the metadata but is present...

Reply:
Probably why I currently remove it from setup.cfg. I must have seen that error some time ago. I guess, that acceptably would avoid the patching then.

Reply:
> I'd rather skip it rather than having a token file.

Done in d85432a569d

Reply:
Let's get this in please.